daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec

```diff
@@ -48,6 +48,8 @@ cp $RPM_BUILD_DIR/%{name}-%{version}/key.pem $RPM_BUILD_ROOT/etc/pki/rhui
 mkdir -p $RPM_BUILD_ROOT/%{_bindir}
 cp $RPM_BUILD_DIR/%{name}-%{version}/rhui-set-release $RPM_BUILD_ROOT/%{_bindir}
 
+# google-rhui-client-update cron job
+cp $RPM_BUILD_DIR/%{name}-%{version}/google-rhui-client-package-update $RPM_BUILD_ROOT/etc/cron.daily
```

Comment:
You need to do the explicit directory creation before the copy
mkdir -p $RPM_BUILD_ROOT/etc/cron.daily

Reply:
done